### ISSUE TITLE
Add search functions

### ISSUE THREAD

**Issue by apachano** [apachano/minetestdb#11]
I want to add searching functions to the website
-Search a list for titles
-Search a list for content in posts
-Probably search other things

**Comment by M3TIOR** [comment]
???

**Comment by apachano** [comment]
Kinda, I am trying to get organized and the issues tab seems to be best for it. Please give feedback if there is a better way for me to do this.

**Comment by M3TIOR** [comment]
I see you've already used a "project", it seems a lot like trello. Why not use that for todos and save the issues tab for problems and bugs. You may also be able to assign project pieces out to me that way.

**Comment by apachano** [comment]
Yea I think I am still going to just use the issues list, it seems more functional. The projects boards do not actually have a comment system unless you make the cards issues which just brings us back here.

**Comment by M3TIOR** [comment]
yuck... God dammit github... Oh well, we could use trello again lol

**Comment by M3TIOR** [comment]
Nah, jkjk, I like a unified system. the less I have to move between the better.
